### ISSUE TITLE
Spotify

### ISSUE THREAD

**Issue by Cheetoman01** [ryannair05/Little12#7]
I love the tweak, but it Broke Spotify-all versions. It just crashes on launch. Thanks!

**Comment by dinouse** [comment]
are you sure it was little12 the imposter?

**Comment by Cheetoman01** [comment]
Absolutely-it was the only tweak I installed. As soon as I uninstalled it, Spotify worked perfectly.



________________________________
